docs/upgradetest.md

```diff
@@ -0,0 +1,42 @@
+This guide is for Harbor upgrade test  
+=======
+
+# Before upgrade
+
+## Prepare data  
+1. At least three users, at least one has admin role.  
+2. Create at least one private and one public project.  
+3. Add several members to project, each role need one.  
+4. Login harbor as project admin, push at least 2 images into project, include signed. pull each image at least one time. 
+5. Login harbor as project developer, push at least 2 images into project, include signed. pull each image at least one time. 
+6. Repeate 4 and 5 to all projects.  
+7. Add at least three endpoints to harbor.  
+8. Add 3 replication rules to each project, first use immediate trigger, second use scheduled, the third use manual, and each rule trigger at least once to generate replication logs.  
+9. Add at least 5 system labels and tag at least one to a tag.  
```

Comment:
Please explicitly document what labels should be added. And what images should be labeled.